website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -145,43 +144,47 @@ Advanced users can configure multiple projects to use the same Cloud CLI executa
 
 <TabItem value="pip" label="Existing dbt Core users (pip)">
 
+If you already have dbt Core installed, you can install the dbt Cloud CLI using pip. Here are some considerations:
+
+- **Prevent conflicts**<br/>
+  To prevent overwriting dbt Core, avoid installing the dbt Cloud CLI with `pip`. Instead, consider using the native installation method (Homebrew) and configure your PATH or create a new virtual environment.<br/>
```

Comment:
I would say this bullet should be "If you want to use pip, create a new virtual environment"